web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/src/ViewsBasicManager.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace Drupal\ys_views_basic;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\Core\Entity\EntityDisplayRepository;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+
+/**
+ * Service for managing the Views Basic plugins.
+ */
+class ViewsBasicManager extends ControllerBase implements ContainerInjectionInterface {
+
+  /**
+   * Allowed entity types for users to select.
+   *
+   * @todo This seems fragile and would better be inside a config page for
```

Comment:
Agreed. This is not sustainable. It could be a field setting or a separate interface. But the goal of this ticket is storage and so I think this is good for what we need at the moment.